bookserver/crud.py

```diff
@@ -494,3 +580,189 @@ async def create_user_chapter_progress_entry(
     async with async_session.begin() as session:
         session.add(new_ucp)
     return UserChapterProgressValidator.from_orm(new_ucp)
+
+
+#
+# Select Question Support
+# -----------------------
+
+
+async def create_selected_question(
+    sid: str,
+    selector_id: str,
+    selected_id: str,
+    points: Optional[int] = None,
+    competency: Optional[str] = None,
+) -> SelectedQuestionValidator:
+    new_sqv = SelectedQuestion(
+        sid=sid,
+        selector_id=selector_id,
+        selected_id=selected_id,
+        points=points,
+        competency=competency,
+    )
+    async with async_session.begin() as session:
+        session.add(new_sqv)
+    return SelectedQuestionValidator.from_orm(new_sqv)
+
+
+async def fetch_selected_question(sid: str, selector_id) -> SelectedQuestionValidator:
```

Comment:
Missing type for `selector_id` -- I assume `str`?

Reply:
Also, I need docs somewhere (perhaps in models) on what/how selectquestions work. What is the `selector_id` (I think this is the `div_id` of the selectquestion) versus the `selected_id` (the `div_id` of the question to select? But don't we allow multiple ids, while this is a single id?)